### PR TITLE
Further reduce VolatilityOracle's timing assumptions, add no-op silo

### PR DESCRIPTION
**Pull request by haydenshively**
Observed a case where available secondsAgo was 60% of what cacheMetadataFor had previously recorded. Not the end of the world because you can always just call cacheMetadataFor again, but to avoid confusing errors I decreased multiplier from 80% to 60%.

Also added no-op silo. Pretty straightforward but we should test on-chain to verify there's no weird interactions with rest of contract

### CODE REVIEW

File: contracts/silos/NOPSilo.sol [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import "contracts/interfaces/ISilo.sol";
+
+contract NOPSilo is ISilo {
+    /// @inheritdoc ISilo
+    string public name;
+
+    constructor(address _token) {
+        name = string(abi.encodePacked(IERC20Metadata(_token).symbol(), " no-op Silo"));
+    }
```
Comment (adamegyed) [inline contracts/silos/NOPSilo.sol:14]: Does it need to be assigned a token? As a no-op silo

Comment (haydenshively) [reply contracts/silos/NOPSilo.sol:14]: Lol I guess not

**Comment by adamegyed** [comment]
Tests also need to be updated with the new scaling.

**Comment by haydenshively** [comment]
> Tests also need to be updated with the new scaling.

It's not that, it's the bytecode. Gotta update it in factory

**Comment by adamegyed** [comment]
Well that's a side effect of the new scaling.... technically

**Comment by haydenshively** [comment]
> Well that's a side effect of the new scaling.... technically

Also apparently you're actually right. Impacts a few constants and (surprisingly) gas in VolatilityOracle tests

**Pull request merged**